src/components/Integrations/IntegrationsList/IntegrationsList.vue

```diff
@@ -162,6 +171,12 @@ export default {
       'hasFeature'
     ]),
 
+    filteredWebhooks: function() {
+      let filteredArray =  this.integrations.filter(x=> !x.customTargets || x.customTargets.length == 0)
+
+      return filteredArray
```

Comment:
in cases like this, you can also just return directly without making a variable: 

```
return this.integrations.filter(x=> !x.customTargets || x.customTargets.length == 0)
```

Reply:
ah yes